base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1836,9 +1843,12 @@ private SingleResponse processRequest(Request req) {
             } else {
                 certStatus = new UnknownInfo();
             }
-        } catch (Exception e) {
+        } catch (EDBRecordNotFoundException e) {
             // not found
-            certStatus = new UnknownInfo(); // not issued not all
+            certStatus = new GoodInfo(); // not issued not all
```

Comment:
despite what's being said in RFC 6960 wrt "good" response, since this is inside the CA, who knows for sure that it is not a cert issued by itself, I think "unknown" is a better fit for this response (v.s. standalone ocsp who has no means of finding out).  In the "note" area of the rfc in that section, it says ' the "unknown" status indicates that the status could not be determined by this responder' so making it "unknown" would be fine I think.

Reply:
Last version of the spec suggest to provide reject status in this case (or good for backward compatibility) so the client does not have to look at other responders. I leave this decision to you.

Reply:
@fmarco76 as a matter of fact, I  had an about face after looking at the rfc more closely.  It should return "revoked" in this case.  However, not having time (minutes away from close of sprint) to find a handy way to construct a "revoked" response at the time, I opted to assign unknown to it.  I think that's what the lab is expecting from the emails I received.  Either way it's still better than returning "good".  This is a rare case anyway, so we can leave it to the lab and patch it if needed.  Thanks!